launch/personal_food_computer_var_types.yaml

```diff
@@ -0,0 +1,82 @@
+recipe_variables:
+  recipe_start:
+    name: recipe_start
+    description: "Represents the start of a recipe"
+  recipe_end:
+    name: recipe_end
+    description: "Represents the end of a recipe"
+environment_variables:
+  air_temperature:
+    name: air_temperature
+    description: "Temperature of the air in degrees Celcius"
+    units: "degrees C"
+  air_humidity:
+    name: air_humidity
+    description: "A measure of the concentration of water in the air relative to the maximum concentration at the current temperature"
+    units: "percent relative"
+  air_carbon_dioxide:
+    name: air_carbon_dioxide
+    description: "The amount of Carbon Dioxide in the air"
+    units: "ppm"
+  air_flush_on:
+    name: air_flush_on
+    description: "Turn on air flush (off by default)"
+    units: "boolean"
```

Comment:
@rwdavis513 in this scenario we use "boolean" for units, but for our other boolean variable (water_level_high) we use no unit. We need to make this consistent.

Suggestion: we should have an additional field called `type` that uses the ROS type keys (`std_msgs/Float32`, etc) and use this to make decisions about how to filter channels.